generator/svelte/src/lib/utils.ts

```diff
@@ -126,6 +126,23 @@ export const isNextIqama = (masjids: [string, IMasjid][], masjidName: string, iq
 	return getNextPrayerForMasjid(masjid).name === iqama;
 };
 
+export const getCurrentPrayer = (masjids: [string, IMasjid][]) => {
+	const sortedPrayers = getSortedPrayers(masjids);
+	return sortedPrayers[sortedPrayers.length - 1];
+};
+
+export const shouldDefaultToJumas = (masjids: [string, IMasjid][]) => {
+	const currentPrayer = getCurrentPrayer(masjids);
+
+	const isThursday = dayjs().day() === EDay.Thursday;
+	const isFriday = dayjs().day() === EDay.Friday;
+
+	const isMaghrib = currentPrayer === EPrayer.Maghrib;
+	const isIsha = currentPrayer === EPrayer.Isha;
+
+	return (isThursday && (isMaghrib || isIsha)) || (isFriday && !isMaghrib && !isIsha);
```

Comment:
Can I ask for a change in the requirements? I found that it is more useful to change the condition to be Friday from Fajr to Asr. In other words, match the condition where the page would normally show next prayer as Zuhr, but because it is Friday, we show the Jumaa layout. Does that make sense?

Reply:
Yes, it makes sense. There is no need to go to the layout once that time has passed. Also, no need to jump to Jumas layout starting from the prior day.

So the way we are going to do them is to show Jumas when the current prayer is Fajr and next is Zuhr.

Reply:
@hammady Updated.

Reply:
Great, thanks!